src/Entity/Item.php

```diff
@@ -77,6 +77,38 @@ public function setId( $id ) {
 	}
 
 	/**
+	 * @param string $languageCode
+	 * @param string $value
+	 *
+	 * @return string
```

Comment:
Uh, I never realized the setters in Entity returned strings. Good you did not copied the `return`code lines. Please remove the `@return`` tags too.